projects/kit/components/input-phone-international/input-phone-international.style.less

```diff
@@ -0,0 +1,60 @@
+@import '@taiga-ui/core/styles/taiga-ui-local';
+
+@flag-width: 1.75rem;
+@flag-height: 1.25rem;
```

Comment:
Not a fan of those variables either, this is not something we will change

Reply:
Fixed in https://github.com/taiga-family/taiga-ui/pull/7580/commits/18f5fea3632817d0f77b2408751721d12509218f